dbt/profiles.yml

```diff
@@ -3,8 +3,9 @@ athena:
   outputs:
     dev:
       type: athena
-      s3_staging_dir: s3://ccao-dbt-athena-test-us-east-1/results/
-      s3_data_dir: s3://ccao-dbt-athena-test-us-east-1/data/
+      s3_staging_dir: s3://ccao-dbt-athena-dev-us-east-1/results/
+      s3_data_dir: s3://ccao-dbt-athena-dev-us-east-1/data/
```

Comment:
I think `ccao-dbt-athena-dev` is clearer than `ccao-dbt-athena-test`, especially in contrast to the `ci` bucket defined below. If you agree, I'll go ahead and delete `ccao-dbt-athena-test` once I merge in this PR.

Reply:
So, just to confirm, the bucket purposes are:

- `s3://ccao-dbt-athena-dev-us-east-1` - For local development. Materialized tables are written to `data/`, where they live in perpetuity.
- `s3://ccao-dbt-athena-ci-us-east-1` - For CI. Same directory structure but things in `data/` are deleted after 30 days
- `s3://ccao-dbt-athena-test-us-east-1` - Now deprecated. Will be deleted.

If so, that's sensible to me 👍